python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/platform.schema.yml

```diff
@@ -182,3 +182,58 @@ keys:
               - str
             min: 1
             max: 128
+          interface:
+            type: dict
+            description: Configure interface related settings for Sfe platform.
+            keys:
+              profiles:
+                type: list
+                primary_key: name
+                description: |-
+                  Configure one or more Receive Side Scaling (RSS) interface profiles.
+                  This is supported on select platforms.
```

Comment:
I would remove this
1/ we have it already on the top level
2/ If we decide to keep it, I don't think the is correct English - maybe `on specific platforms` would be more correct

```suggestion
```